application/src/main/java/md/maib/retail/application/delete_campaign/DeleteCampaign.java

```diff
@@ -0,0 +1,23 @@
+package md.maib.retail.application.delete_campaign;
+
+import am.ik.yavi.builder.ValidatorBuilder;
+import am.ik.yavi.core.Validator;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+import md.maib.retail.application.register_newcampaign.RegisterCampaign;
+import md.maib.retail.model.campaign.CampaignId;
+
+@Getter
+@RequiredArgsConstructor
+@ToString
+public class DeleteCampaign {
```

Comment:
must be static factory that create command and call validator.
constructor must be private
class must be final

Reply:
how we will use create method from Validator in service?

or it will used only in Rest?